guides/advanced/building-sytems.md

```diff
@@ -0,0 +1,349 @@
+
+1. **Modify Buildroot Configuration**
+   Nerves systems use Buildroot for building firmware. The `make menuconfig` command opens a menu-based interface where you can modify the Buildroot configuration:
+
+   ```bash
+   cd o/<system short name>
+   make menuconfig
+   ```
+
+   In this interface, you can:
+
+   - Add or remove packages.
+   - Configure kernel options.
+   - Set custom build flags.
+
+   > **Tip:** Only make changes you understand, as incorrect settings may cause build failures or unstable firmware. For more details on Buildroot configuration, refer to the [Buildroot user manual](https://buildroot.org/downloads/manual/manual.html).
+
+2. **Save the Updated Configuration**
+   After making changes in `menuconfig`, save the configuration back to the system’s default configuration file (`nerves_defconfig`) using:
+
+   ```bash
+   make savedefconfig
+   ```
+
+   This ensures that your changes are preserved in the Buildroot configuration and can be reused in future builds. Learn more about Nerves system configuration in the [Nerves documentation](https://hexdocs.pm/nerves/systems.html).
+
+3. **Rebuild the System**
+   To apply your changes, clean the output directory for the system and rebuild:
+
+   ```bash
+   rm -rf o/<system short name>
+   mix ns.build
+   ```
+
+   This ensures a fresh build with your updated configuration.
+
+4. **Make Additional Modifications (Optional)**
+   You can further customize the Nerves system by modifying other configuration files, such as:
+
+   - **Linux kernel configuration:** Located in the Buildroot environment.
+   - **System files:** Add or update scripts, binaries, or other files required by your application.
+
+   To dive deeper into kernel customization, see the [Linux Kernel Documentation](https://www.kernel.org/doc/html/latest/).
+
+5. **Test the Custom Build**
+   After rebuilding, test the custom firmware on your hardware to ensure it meets your requirements. If issues arise:
+
+   - Review the Buildroot logs in `o/<system short name>/build/`.
+   - Iterate on the configuration as needed.
+
+6. **Version Control Your Changes**
+   If your customizations are for long-term use, consider committing your changes to version control. This is especially useful for:
+
+   - Collaborating with other developers.
+   - Reproducing builds in the future.
+
+   Example:
+
+   ```bash
+   git add config/nerves_defconfig
+   git commit -m "Customize Buildroot configuration for <system name>"
+   ```
+
+---
+
+## Conclusion
+
+The `nerves_systems` repository enables an alternative, faster approach to building Nerves systems, especially for developers working extensively with Buildroot or maintaining multiple systems. By following this guide, you can create and customize systems for your hardware platform, unlocking new possibilities within the Nerves framework.
```

Comment:
I don't think we need a conclusion section for the guide since it's part of the main docs.